src/doc/python_scripting/functions.rst

```diff
@@ -1510,38 +1510,46 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineArrayExpression function returns 1 on success and 0 on failure.
 
 
 **Description:**
 
     DefineArrayExpression creates new array variables.
-    Expression variables can be plotted like any other variable.
-    The variableName argument is a string that contains the name of the new
-    variable. You can pass the name of an existing expression if you want
-    to provide a new expression definition.
-    The expression argument is a string that contains the definition of the
-    new variable in terms of math operators and pre-existing variable names
-    Reference the VisIt User's Manual if you want more information on
-    creating expressions, such as expression syntax, or a list of built-in
-    expression functions.
+    Array variables are a collection of scalar variables that are grouped together.
+    Currently there are no plots that take array variables, but they can be used in queries, such as the "XRay Image" query or used as a pick variable that creates a Histogram plot of the array variable for a specific zone.
```

Comment:
Label plot takes an array variable.

Also, do array variables require all components to have same centering? I think so. If true, might be worth a remark here about that and link to [expression compatablity gotchas](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/using_visit/Quantitative/Expressions.html#expression-compatibility-gotchas)

Reply:
Yes, they have to have the same centering. I added a sentence to that effect. I'm not sure I want to link to the gotchas, since that is relevant to all expressions and I already link to two sections of the expression grammar section in the standard boiler plate for all the DefineXXXExpression functions. If you think I should add it to all of them let me know and I'll make it so.

Reply:
Added a link to the expression compatibility gotchas to each section of DefineXXXExpression functions where appropriate.